examples/adder/tests/Makefile

```diff
@@ -37,7 +37,7 @@ else
 WPWD=$(shell pwd)
 endif
 
-PYTHONPATH := $(PWD)/../model:$(PYTHONPATH)
+export PYTHONPATH := $(PWD)/../model:$(PYTHONPATH)
```

Comment:
It's very tempting to remove this in a future patch and just use `import model.something` in the test.

Reply:
Agree. Much nicer.  In the test, one can append to `sys.path` also rather than this. I can do in this PR?

Reply:
CI has already passed here, may as well merge this one first.